examples/server_client.rs

```diff
@@ -0,0 +1,100 @@
+//! Note that the terms "client" and "server" here are purely what we logically associate with them.
+//! Technically, they both work the same.
+
+extern crate laminar;
+
+use std::{
+    io::stdin,
+    net::{SocketAddr, ToSocketAddrs},
+};
+
+use laminar::{
+    error::Result,
+    Packet,
+    UdpSocket,
+};
+
+const SERVER: &str = "localhost:12351";
+
+fn addr(s: &str) -> Result<SocketAddr> {
+    s.to_socket_addrs().map_err(Into::into).map(|mut x| x.next().unwrap())
+}
+
+fn server() -> Result<()> {
+    let mut socket = UdpSocket::bind(SERVER)?;
+
+    loop {
+        match socket.recv()? {
+            Some(packet) => {
+                let msg = packet.payload();
+
+                if msg == b"Bye!" {
+                    break;
+                }
+
+                let msg = String::from_utf8_lossy(msg);
```

Comment:
Why not check here if message is `Bye!` with `msg`.

Reply:
doesn't really matter I guess?

Reply:
Yes but you convert to times bytes to string. Just a little thing.

Reply:
Sorry, I don't understand. I just did it that way because checking for that byte sequence seemed more accurate than checking a lossy conversion that was intended only for printing. Of course it'll be the same this case, that's just why I did it if that makes any sense.

Reply:
Oke lets leaf it like that.